docs/writing_a_parser_plugin/parser_shell.md

```diff
@@ -0,0 +1,128 @@
+# Creating the shell for your parser
+
+First, create a directory for your parser under the relevant parser project directory:
+
+    cd dependencies/parsers/<parserproject>
+    mkdir <parsername>
+
+In the following please note the naming conventions:
+
+- <parsername\> - flat case, e.g., quantumespresso
+
+- <ParserName\> - Pascal case, e.g., QuantumEspresso
+
+- <parser_name\> - snake case, e.g., quantum_espresso
+
+- <PARSERNAME\> - upper flat case, e.g., QUANTUMESPRESSO
```

Comment:
Pls state which convention is used where, i.e. filenames, classes, etc.

Reply:
made a TODO